pkg/ccl/backupccl/backup_planning.go

```diff
@@ -332,12 +333,16 @@ func resolveOptionsForBackupJobDescription(
 		newOpts.EncryptionPassphrase = tree.NewDString("redacted")
 	}
 
-	for _, uri := range kmsURIs {
-		redactedURI, err := cloud.RedactKMSURI(uri)
-		if err != nil {
-			return tree.BackupOptions{}, err
-		}
-		newOpts.EncryptionKMSURI = append(newOpts.EncryptionKMSURI, tree.NewDString(redactedURI))
+	var err error
+	// TODO(msbutler): use cloud.RedactKMSURI(uri) here instead?
```

Comment:
Does RedactKMSURI differ from the driver in sanitizeURIList? Just want to make sure we are not introducing a regression by switching the redaction method?